routes/web.php

```diff
@@ -16,3 +16,4 @@
 */
 
 Route::get('cities', [CityController::class, 'index']);
+Route::post('cities/create', [CityController::class, 'store']);
```

Comment:
you can think of grouping routes with same prefixes. As you can see here, 'cities/' repeats many times here. I can give you an example via slack!